packages/react-inbox/src/components/Messages2.0/actions/index.tsx

```diff
@@ -78,32 +77,23 @@ const MessageActions: React.FunctionComponent<{
 
   const handleEvent = (eventName: string) => (event: React.MouseEvent) => {
     event?.preventDefault();
+    if (!messageId) {
```

Comment:
what are situations where we won't have messageId?

Reply:
i think we should always have it, let me verify